.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
```

Comment:
- [ ] Kindly take a look at the updated readme file. You can find it [here](https://github.com/microverseinc/readme-template). I will love to know if you don't like it compared to the current one you are using. This is because the updated one is so detailed and user-friendly.